### PR TITLE
Add `Status` dropdown to `SearchForm` component

### PR DESCRIPTION
**Pull request by reyronald**
**Release Type:** *Non Breaking Feature* 

Fixes https://x-team-internal.atlassian.net/browse/XP-2303

## Description

What the title says.

Tested on Chrome, Firefox, Edge and integrated into `auto`. Seems to be working fine in all cases.

**I can't access the Sketch file from the JIRA ticket though so there might be some misalignment in the styles :(. Would appreciate it if any reviewer could check.**

**Also, if anyone thinks of a better approach or notices something missing don't hesitate to comment. CSS and styling is not my forte :P**

## Checklist

**Before submitting a pull request,** please make sure the following is done:

- [x] set yourself as an assignee
- [x] set appropriate labels for a PR (initially it's mostly `[zube]: In Review`)
- [x] set `[zube]: In Review` label for respective issue as well
- [x] make sure your code lints (`npm run lint`)
- [x] Flow typechecks passed (`npm run typecheck`)
- [x] my change requires a change to the component's documentation (`.snap`)
- [x] component's documentation (`.snap`) is changed or added accordingly
- [x] if you've fixed a bug or added code that should be tested, add unit tests
- [x] if any snapshots have been changed, verify that component still works and looks as expected and update changed snapshot
- [x] **manually tested the component** by running it in the browser and checked nothing is broken and operates as expected!

## Related PRs

branch | PR
------ | ------
XP-2303-add-status-field-to-search-form | [Auto](https://github.com/x-team/auto/pull/1598)
XP-2303-add-status-field-to-search-form | [Auto API](https://github.com/x-team/auto-api/pull/534)

## Impacted Areas in Application

- The `SearchForm` component

## Screenshots

List view:

list

Tabular view:

tabular

### CODE REVIEW

File: src/components/ui/SearchForm.js [hunk 1]
```diff
@@ -64,15 +65,23 @@ const listTheme = {
...
-    padding: 20px 30px 0
-    border-top: 1px solid ${theme.lineSilver4}
+    padding: 0 30px
     box-sizing: border-box
   `),
 
+  selectFields: cmz(`
+    display: inline-block
+    width: 50%
+    margin-right: 16px
```
Comment (migueliriano) [inline src/components/ui/SearchForm.js]: Can you confirm if need to use `rem` instead of `px` here, please?

Comment (nicksp) [reply src/components/ui/SearchForm.js]: @migueliriano We decided to use `rem` as the `font-size` for all elements on the page, including the margin and padding for texts.

Comment (nicksp) [reply src/components/ui/SearchForm.js]: for boxes/containers we use fixed sizing (e.g. width, height, margin and padding) cc/ @bernardodiasc

Comment (bernardodiasc) [reply src/components/ui/SearchForm.js]: We should use only for typography elements and properties. So if the margin is applied in a paragraph for instance, the relative unit can be used. But for elements meant for layout purposes we need to stick to absolute units. :)

Comment (nicksp) [reply src/components/ui/SearchForm.js]: Seems like what I said, right 😄 ?

Comment (nicksp) [reply src/components/ui/SearchForm.js]: Just to confirm.

Comment (nicksp) [reply src/components/ui/SearchForm.js]: tagging @bernardodiasc

Comment (bernardodiasc) [reply src/components/ui/SearchForm.js]: > Seems like what I said, right 😄 ?

yeah, that's right. I didn't saw your messages when I wrote my reply

Comment (nicksp) [reply src/components/ui/SearchForm.js]: no probs man 😆 Just wanted to double check.

**Comment by nicksp** [comment]
@reyronald Is it in progress or in review 😄 ?

**Comment by reyronald** [comment]
Apparently Github is somewhat back to normal. This is In Review.

**Comment by migueliriano** [comment]
@reyronald - can we resolve the branch conflic?

**Comment by nicksp** [comment]
@migueliriano I resolved it. Just run UI page and updated snapshots

image

**Pull request merged**